scripts/azure-linux_mac-release.yml

```diff
@@ -103,47 +106,47 @@ steps:
     make -j4
     make -C tiledb install
   displayName: 'Build libtiledb'
+  continueOnError: true
```

Comment:
Leftovers to remove?

Reply:
@ihnorton I made this change on purpose so if the build fails we'll now tar and upload the build dir artifact so we can debug it. The release artifact will still fail and won't upload. The github release stage also will not run.

Reply:
Got it, thanks